### PR TITLE
Add automation to Channel Protection Amulet

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/equipment/channel-protection-amulet.json [hunk 1]
```diff
@@ -37,7 +37,32 @@
...
+                                    "self:mode:undead",
+                                    "item:slug:heal"
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "key": "Resistance",
+                "label": "PF2E.SpecificRule.Equipment.ChannelProtectionAmuletLabel",
+                "type": "custom",
```
Comment (SpartanCPA) [inline packs/equipment/channel-protection-amulet.json:62]: Is this just so it works against either? I should be Vitality and Void I'd imagine

Comment (TikaelSol) [reply packs/equipment/channel-protection-amulet.json:62]: It's not general resistance against either, only resistance to heal or harm.

File: packs/equipment/channel-protection-amulet.json [hunk 2]
```diff
@@ -37,7 +37,32 @@
...
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
+                    {
+                        "or": [
+                            {
+                                "and": [
+                                    "self:mode:living",
+                                    "item:slug:harm"
```
Comment (stwlam) [inline packs/equipment/channel-protection-amulet.json:48]:
```suggestion
                                    "item:type:spell",
                                    "item:slug:harm"
```

File: packs/equipment/channel-protection-amulet.json [hunk 3]
```diff
@@ -37,7 +37,32 @@
...
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "self:mode:undead",
+                                    "item:slug:heal"
```
Comment (stwlam) [inline packs/equipment/channel-protection-amulet.json:54]:
```suggestion
                                    "item:type:spell",
                                    "item:slug:heal"
```

**Pull request merged**